Source/EasyTipView.swift

```diff
@@ -46,12 +46,24 @@ public class EasyTipView: UIView, Printable {
         public var font                    :   UIFont?
         public var textAlignment           :   NSTextAlignment
         
+        public var arrowHeight             :   CGFloat
```

Comment:
This would become duplicates with `Constants`. I believe I came up with a cleaner way to make those things customizable: https://github.com/teodorpatras/EasyTipView/pull/12